charts/velero/values.yaml

```diff
@@ -72,6 +72,12 @@ metrics:
 # Install CRDs as a templates. Enabled by default.
 installCRDs: true
 
+# Enable/disable helm hooks annotations for Helm v2 in CRDs and/or templates
```

Comment:
I think it's not specific on helm v2 only.

Reply:
ArgoCD uses only Helm v2 until v1.4. Starting in v1.5 it has both v2 and v3, using v3 when Chart.yaml apiVersion is v2.

Reply:
okay, since we're using `apiVersion: v1` https://github.com/vmware-tanzu/helm-charts/blob/c90e413/charts/velero/Chart.yaml#L1, so the ArgoCD would use helm v2 behavior even we use ArgoCD version >= 1.5, right?

Reply:
That’s right!

Reply:
> Not exactly, if you want to provide your own values.yaml for this chart for the helm install you need to build your own chart which has this one as a dependency. Which results in usually helm v3 being the actual engine.
This is because argocd requires the source of both helm chart and altered values.yaml to be in the same repository.

I use it that way, with velero chart as a dependency, and ArgoCD also uses Helm v2. For Helm v3 I have to change Chart.yaml apiVersion of the chart that I build (the one which imports velero chart) to v2, or force ArgoCD to use Helm v3 [as described here](https://argoproj.github.io/argo-cd/user-guide/helm/#helm-version).